.cicd/pipeline.yml

```diff
@@ -425,8 +425,22 @@ steps:
       PKGTYPE: "rpm"
     agents:
       queue: "automation-eks-eos-tester-fleet"
+    key: "amazonlinux2pb"
     timeout: ${TIMEOUT:-10}
-    skip: ${SKIP_AMAZON_LINUX_2}${SKIP_PACKAGE_BUILDER}
+    skip: ${SKIP_AMAZON_LINUX_2}${SKIP_LINUX}${SKIP_PACKAGE_BUILDER}
+
+  - label: ":aws: Amazon_Linux 2 - Test Package"
```

Comment:
I still think we should call this `Install Package` so it is more obvious what is going on to the customer.